src/main/java/frc/robot/Main.java

```diff
@@ -1,10 +1,3 @@
-/*----------------------------------------------------------------------------*/
```

Comment:
I think you need to leave the FIRST copyright

Reply:
Just realized that this was in a lot of files.  How about leaving the FIRST copyright and adding a 1073 copyright header?  Maybe something like:
All changes from FIRST/WPLIB original code is copyright FRC Team 1073 and is covered by the same licensing as the FIRST BSD license